### PR TITLE
demo/synronized-charts-tweaks

### PR DESCRIPTION
**Pull request by Xillez**
Added some minor tweaks to synchronized charts demo.

---

Asana task: https://app.asana.com/0/1204848917817812/1207412623292841/f

### CODE REVIEW

File: samples/highcharts/demo/synchronized-charts/demo.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
         where multiple charts are often used to visualize related
         information.
     </p>
-    <a href="https://www.highcharts.com/demo/dashboards/sync">
+    <a href="https://www.highcharts.com/demo/dashboards/sync" target=_top>
```
Comment (bre1470) [inline samples/highcharts/demo/synchronized-charts/demo.html:14]: Double quote characters missing.

**Review by bre1470 — APPROVED** [review]
One typo, otherwise looking good.

**Review by hubertkozik — APPROVED** [review]
I agree with Sophie's comment, besides that LGTM.

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**